edgio.config.js

```diff
@@ -1,24 +1,59 @@
 // This file was automatically added by edgio init.
 // You should commit this file to source control.
-// Learn more about this file at https://docs.edgio.co/guides/edgio_config
+// Learn more about this file at https://docs.edg.io/guides/edgio_config
 module.exports = {
-  routes: './edgio/routes.js',
-  connector: './edgio',
+  connector: '@edgio/vue-cva',
+
+  // The name of the team in Edgio to which this app should be deployed.
+  // team: 'my-team-name',
+
   backends: {
     // Define a domain or IP address to proxy as a backend
-    // More on: https://docs.edgio.co/guides/edgio_config#backends
+    // More on: https://docs.edg.io/guides/layer0_config#backends
     api: {
-      domainOrIp: 'edgio-docs-edgio-ecommmerce-api-example-default.edgio-limelight.link',
-      hostHeader: 'edgio-docs-edgio-ecommmerce-api-example-default.edgio-limelight.link',
```

Comment:
this was probably caused by global string replace, as the url is invalid -> deployments fail, so no new version is deployed, therefore it looks like it works

Reply:
Thank you, yes!